hveto/utils.py

```diff
@@ -62,3 +69,58 @@ def channel_groups(channellist, ngroups):
     n = int(ceil(len(channellist) / ngroups))
     for i in range(0, len(channellist), n):
         yield channellist[i:i+n]
+
+
+def primary_vetoed(starttime=None, hveto_path=None, snr=6.0,
+                   significance=5.0):
+
+    """Catalogue all vetoed primary triggers from a given analysis
+
+    This utility queries the output of an hveto analysis for the triggers
+    vetoed from its primary channel over all rounds (up to thresholds on
+    signal-to-noise ratio and round significance).
+
+    Parameters
+    ----------
+    starttime : `str` or `float`
+        start GPS time for this analysis
+
+    hveto_path : 'str'
+        path of the hveto files directory, not required if starttime given
```

Comment:
To ensure good rendering in the documentation, I suggest using Markdown rendering around the `starttime` variable:
```suggestion
        path of the hveto files directory,
        not required if ``starttime`` is given
```